packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactEditTextInputConnectionWrapper.kt

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.react.views.textinput
+
+import android.view.KeyEvent
+import android.view.inputmethod.InputConnection
+import android.view.inputmethod.InputConnectionWrapper
+import com.facebook.react.bridge.ReactContext
+import com.facebook.react.uimanager.UIManagerHelper
+import com.facebook.react.uimanager.events.EventDispatcher
+
+/**
+ * A class to implement the TextInput 'onKeyPress' API on android for soft keyboards. It is
+ * instantiated in {@link ReactEditText#onCreateInputConnection(EditorInfo)}.
```

Comment:
The way how you reference in KDoc is different than in Javadoc. E.g:

```suggestion
 * instantiated in [ReactEditTex.onCreateInputConnection].
```

Could you please apply KDoc format to this one and the other links as well?

Reply:
Done. Also moved some newlines around.